src/pages/tutorial/task/task.js

```diff
@@ -94,6 +96,48 @@ class TaskPage extends React.Component {
     setProgress(progress);
   };
 
+  areDocLinksReady = () => {
```

Comment:
@dlabaj I've added this in as a temporary fix for the Asciidoc not re-rendering when I update the attributes. I'll look into why the Asciidoc isn't re-rendering.